topics/a11y/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: A11y (Accessibility)
```

Comment:
❤️ Thanks so much for the contribution! I'm wondering if the root should actually be `Web Accessibility`, with the `a11y` as an alias. What do you think @shaedrich

Reply:
❤️ Anytime, I was wondering the same thing, among other option but that one appears to be the most complete solution after all&mdash;I gladly chance it. One sec.